source/github-events/on-fragment-load.ts

```diff
@@ -6,7 +6,7 @@ const getDeduplicatedHandler = mem((callback: EventListener): DelegateEventHandl
 	event.delegateTarget.addEventListener('load', callback);
 });
 
-export function createFragmentLoadListener(fragmentSelector: string, callback: EventListener, signal: AbortSignal): void {
+function createFragmentLoadListener(fragmentSelector: string, callback: EventListener, signal: AbortSignal): void {
```

Comment:
This whole file will be hard to drop 🥲 
These functions are used about 10 times total, which means a lot of refactoring 😬